### PR TITLE
HTS-519:  Fix the missing continueURL param in the IV result

### PR DESCRIPTION
**Pull request by sureshhmrc**

### CODE REVIEW

File: app/uk/gov/hmrc/helptosavefrontend/controllers/IvController.scala [hunk 1]
```diff
@@ -45,6 +45,7 @@ class IvController @Inject() (val sessionCacheConnector: SessionCacheConnector,
     //Will be populated if we arrived here because of an IV success/failure
     val journeyId = request.getQueryString("token").orElse(request.getQueryString("journeyId"))
     val allowContinue = true
+    val newIvUrl = ivUrl(continueURL)
```
Comment (MarcusQA) [inline app/uk/gov/hmrc/helptosavefrontend/controllers/IvController.scala]: Just a question - does "new" in the variable name convey what the variable is doing?

Comment (sureshhmrc) [reply app/uk/gov/hmrc/helptosavefrontend/controllers/IvController.scala]: not like that - i meant to create . a new iv url val here...

Comment (sureshhmrc) [reply app/uk/gov/hmrc/helptosavefrontend/controllers/IvController.scala]: i used the name as we already have a method name `ivUrl(....)` so it not allow me to write 
`val ivUrl = ivUrl(continueURL)` (edited)

**Comment by AndyHWChung** [comment]
LGTM

**Pull request merged**